### PR TITLE
Add `on_break?` schedule method

### PR DESCRIPTION
**Pull request by craiglittle**
Similar to `on_holiday?` with holidays, this method can be used to determine if a provided time occurs during a break.

@zendesk/darko

### CODE REVIEW

File: spec/calculation/on_break_spec.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    context 'when the time is at the end of a break' do
+      let(:time) { Time.utc(2006, 1, 4, 13) }
+
+      it 'returns false' do
+        expect(calculation.result).to eq false
+      end
+    end
+
+    context 'when the time is not on a break' do
```
Comment (alex-stone) [inline spec/calculation/on_break_spec.rb]: **nit**: This is fine, but I prefer `when the time is not during a break` for readability purposes.

**Comment by alex-stone** [comment]
:+1:, with or without suggested spec wording changes.

File: spec/calculation/on_break_spec.rb (hunk 1 above)
Comment (craiglittle) [inline spec/calculation/on_break_spec.rb]: Yeah, `on a` does match up with the method name (`on_break?`) better, but it is also nice to provide richer/different descriptions in the docstrings to flesh out the intention. Will change!

**Comment by craiglittle** [comment]
@alex-stone :haircut:

**Comment by alex-stone** [comment]
:+1:

**Pull request merged**